Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM fedora:27
+
+RUN dnf install -y python2 python2-junit_xml beaker-client \
+	bison flex python2-mock dnf-plugins-core
+RUN dnf builddep -y kernel
+
+# Use a default checkout of the kernel as our workdir Also make it the default directory
+# and setup an environment variable to tell the skt executable to use this workdir
+#
+# Notice that we give 777 and 666 permissions on this directory. This is because Openshift
+# likes to run containers as random UIDs, and we cannot rely on SKT being root
+RUN git clone git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git /var/work && \
+	chmod -R ugo+w /var/work
+ENV WORKDIR=/var/work
+WORKDIR /var/work
+
+# Pull a default kernel config down for use in the container
+# It's expected that this will be overriden either by mounting a
+# new config file, or by adding one in a layered container
+RUN mkdir -p /var/config && \
+        git clone --depth=1 https://src.fedoraproject.org/git/rpms/kernel.git /tmp/kernel && \
+        mv /tmp/kernel/*.config /var/config/ && \
+        rm -rf /tmp/kernel
+ENV BASECONFIG=/var/config/kernel-x86_64.config
```

Comment:
Do you think you can pull the config directly from `https://src.fedoraproject.org/rpms/kernel/raw/master/f/kernel-x86_64.config` with wget/curl?

Reply:
Done.